xdggs/h3.py

```diff
@@ -22,6 +21,32 @@
 from xdggs.utils import _extract_cell_id_variable, register_dggs
 
 
+def polygons_shapely(wkb):
+    import shapely
+
+    return shapely.from_wkb(wkb)
+
+
+def polygons_geoarrow(wkb):
+    import shapely
+    from arro3.core import list_array
+
+    polygons = shapely.from_wkb(wkb)
+    _, coords, (geom_offsets, ring_offsets) = shapely.to_ragged_array(polygons)
+
+    polygon_array = list_array(geom_offsets, list_array(ring_offsets, coords))
+    polygon_array_with_geo_meta = polygon_array.cast(
+        polygon_array.field.with_metadata(
+            {
+                "ARROW:extension:name": "geoarrow.polygon",
+                "ARROW:extension:metadata": '{"crs": "epsg:4326"}',
```

Comment:
Technically this should be PROJJSON as of the current spec: https://github.com/geoarrow/geoarrow/blob/main/extension-types.md#extension-metadata

But there's discussion to loosen the spec https://github.com/geoarrow/geoarrow/pull/40

Reply:
that's good to know. I've switched to using `pyproj.CRS.from_epsg(4326).to_json()` for now, but I definitely support abbreviating that field, PROJJSON is so much longer than the epsg string.